frontend/pages/ManageControlsPage/OSUpdates/components/TargetSection/TargetSection.tsx

```diff
@@ -109,7 +109,8 @@ const TargetSection = ({
   });
 
   const renderTargetForms = () => {
-    if (isMacMdmEnabled && isWindowsMdmEnabled) {
+    if (!isMacMdmEnabled && !isWindowsMdmEnabled) {
```

Comment:
Was this just for testing? Do we want to restore the original?

Reply:
yes omgg ty